cypress/integration/control/intersection.spec.js

```diff
@@ -0,0 +1,35 @@
+const FORCE = {force: true}
+
+describe('Intersect control', function () {
+  beforeEach(function () {
+      cy.visit('/');
+  
+      // Draw polygon
+      cy.get('[title="Draw Polygon"]').click();
+      cy.get('.ol-overlaycontainer').click(500, 200, FORCE);
```

Comment:
Can you describe where do you click if it's on the map on an HTMl element

Reply:
Done